generator/src/test/java/com/scottlogic/deg/generator/cucumber/ContainingRegex.feature

```diff
@@ -291,12 +290,17 @@ Scenario: Running a 'containingRegex' request alongside a non-contradicting matc
        | null |
        | "bb" |
 
-Scenario: Running a 'containingRegex' request alongside a contradicting matchingRegex constraint should fail with an error message
+Scenario: Running a 'containingRegex' request alongside a contradicting matchingRegex constraint should be successful
```

Comment:
for consistency should this test be "should generate null"?

Reply:
Yes, I agree, test name updated